tests/test_exporters_jaeger.py

```diff
@@ -38,57 +38,57 @@ def setUp(self):
         self._test_span.start()
         self._test_span.end()
 
-        self.connection_patcher = mock.patch("http.client.HTTPConnection")
+        self.connection_patcher = patch("http.client.HTTPConnection")
         self.connection_mock = self.connection_patcher.start()
         conn = self.connection_mock.return_value
-        response = http.client.HTTPResponse(mock.Mock())
+        response = http.client.HTTPResponse(Mock())
         response.status = 200
         conn.getresponse.return_value = response
 
     def tearDown(self):
         self.connection_patcher.stop()
 
+    @patch.dict(os.environ, {"OTEL_TRACES_EXPORTER": "jaeger-thrift-splunk"})
     def test_exporter_uses_collector_not_udp_agent(self):
-        exporter = _new_jaeger_exporter(
-            Options(
-                endpoint=self.endpoint,
-                resource_attributes={"service.name": self.service_name},
-            )
-        )
-        agent_client_mock = mock.Mock(spec=jaeger_exporter.AgentClientUDP)
+        options = Options()
+        exporter = options.span_exporter_factories[0](options)
```

Comment:
`options.span_exporter_factories[0](options)` 

the fact that we need to pass the `options` at the end feels wrong. Maybe we can pass somehow the `option` when creating the `partial` ?